### PR TITLE
PRWP TTL training

### PR DESCRIPTION
**Pull request by kbjarkefur**
These slides renders in Quarto

### CODE REVIEW

**Comment by mariaruth** [comment]
@kbjarkefur these look good. a few comments:

- my general comment is that we want to make this a little bit more workflow-focused than it is at the moment. I think one way to do that would be to make up a project and give an example for each section or to choose a real project with a public repo and add a demo (either broken by section or all at once at the end). the latter option is probably better if you can find a good example. there's always pink cars!

- it would be helpful to add a motivation slide at the beginning, similar to the blurb you wrote up. maybe the xkcd version control cartoon or similar. let's take the chance to catch people's attention, especially TTLs, and reinforce why this session would be useful to them (and that they should keep paying attention!)

- slide 7, suggest making more explicit the point that this saves all previous versions without actually saving them as separate files on your own computer (avoiding messy shared folders and confusion from TTL as to what is the right version to look at)

- slide 9, the second point about emails is a useful one (it's the kind of point that i was suggesting for commits) but is incomplete (i assume you meant to say makes this kind of email unnecessary

- slide 10, i would highlight this for TTL audience. this is 99% of what i use these days, for example. it's super helpful as a TTL to be able to see exactly what has changed. if i ask an RA to change a specification, it's easy to see through the PR exactly which lines of codes have changed and whether it's now coded in the way i wanted. this is particularly helpful with long dofiles

- git clients -- i would add the earlier slide about git/github and email analogy to this section. also suggest moving it to the end, it's good to know but i think lower priority than the reproducibility features

- slide 20, this seems a bit redundant with the content on what files should go on git

File: GitHub-trainings/git-github-ttl-intro/template.qmd [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+
+```{r Setup}
+#| include: false
+
+# DIME template:
+source("_extensions/dime/setup_dime_palettes.R")
+source("_extensions/dime/setup_ggplot2_dime.R")
+```
+
+# Motiviation and fundamentals
```
Comment (luisesanmartin) [inline GitHub-trainings/git-github-ttl-intro/template.qmd]:
```suggestion
# Motivation and fundamentals
```
there's a typo here

File: GitHub-trainings/git-github-ttl-intro/template.qmd [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+        <div style="text-align: justify;">
+          Have multiple roots in your projects file paths. Keep data in the old location and then point data file paths to a data folder in a that location. Point code file paths to the Git folder.<br><br>If intermediate data is saved to the Git folder, make sure that data folders or data formats are ignored.
+        </div>
+      </td>
+    </tr>
+  </tbody>
+</table>
+```
+
+# Reproducibilty featueres
```
Comment (luisesanmartin) [inline GitHub-trainings/git-github-ttl-intro/template.qmd]:
```suggestion
# Reproducibilty features
```
typo

File: GitHub-trainings/git-github-ttl-intro/template.qmd [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+# Git Clients
+
+## GitHub Desktop
+
+:::: {.columns .v-center-container}
+::: {.column width="50%"}
+
+A Git Client is a software that allows you to package your edits into commits. Many alternatives exists.
+
+We strongly recommend everyone using a graphic Git Client such as [GitHub Desktop](https://desktop.github.com/). Unless you are a computer scientists do not use the CLI tool until you are an advanced Git user.
```
Comment (luisesanmartin) [inline GitHub-trainings/git-github-ttl-intro/template.qmd]: I suggest mentioning here when presenting (not necessarily in slides) that it doesn't need admin approval/rights for installation in WB computers or any computer

File: GitHub-trainings/git-github-ttl-intro/template.qmd [hunk 4]
```diff
@@ -0,0 +1,410 @@
...
+
+# Reproducibilty featueres
+
+## Version control raw text outputs
+
+
+:::: {.columns .v-center-container}
+::: {.column width="50%"}
+
+Tables in `.tex`, `.csv`, or `.md` format works perfectly to version control on Git
```
Comment (luisesanmartin) [inline GitHub-trainings/git-github-ttl-intro/template.qmd]:
```suggestion
Tables in `.tex`, `.csv`, or `.md` format works perfectly to version control results on Git
```
suggestion for clarity

File: GitHub-trainings/git-github-ttl-intro/template.qmd [hunk 5]
```diff
@@ -0,0 +1,410 @@
...
+
+This allows you to be fully aware of if any changes in your code makes any difference to your outputs
+
+:::
+
+::: {.column width="50%"}
+TODO: Screen shot of latex table with outputs that has changed
```
Comment (luisesanmartin) [inline GitHub-trainings/git-github-ttl-intro/template.qmd]: would be also nice to add a screenshot of a PNG image comparison. That shows a nice feature of GH desktop that would be attractive for TTLs

**Review by luisesanmartin — COMMENTED** [review]
Left some suggestions and a couple of typo corrections but nothing really major. I think this is a good general structure for the presentation and I suggest that the pending screenshots/images emphasize what are the features of git/GH that facilitate code dev compared to what Dropbox and OneDrive offer. My guess is that we're gonna get questions of the type "but how is this really better than Dropbox if that's automated?" other than the argument that changes in Dropbox are eventually lost

**Comment by kbjarkefur** [comment]
Fixes #25

File: GitHub-trainings/git-github-ttl-intro/template.qmd (hunk 5 above)
Comment (kbjarkefur) [reply GitHub-trainings/git-github-ttl-intro/template.qmd]: done in d15cfd0449be0eed47359fca473aeefb1967d744

**Pull request merged**